### DISCUSSION TITLE
How to add canada websites

### DISCUSSION THREAD

**Discussion by gkollaba** [jef/streetmerchant#2318] [Help]
Hi, I am new to this. I wanted to know how to make the bot go to canada websites and not US?

**Comment by 0000day** [comment]
## Step 1
Your first point of interest would be [src\store\model](https://github.com/jef/streetmerchant/tree/main/src/store/model).
You would have to create a new model for the new website. If you are unfamiliar with coding, just copy the content of a model (like `alternate.ts`), create a new file for your model and paste it.

Then go step by step, lookup every variable in there and start replacing,

## Step 2
Open [src\store\model\index.ts](https://github.com/jef/streetmerchant/blob/main/src/store/model/index.ts) and lookup where `Alternate` occurs (e.g. the import on top and the Map-entry at 155). Then go with your new model and add similar entries there.
